diff -Naur node-v6.10.0/deps/npm/lib/utils/map-to-registry.js node-v6.10.0b/deps/npm/lib/utils/map-to-registry.js
--- node-v6.10.0/deps/npm/lib/utils/map-to-registry.js	2017-02-21 18:40:14.000000000 -0600
+++ node-v6.10.0b/deps/npm/lib/utils/map-to-registry.js	2017-04-18 14:38:29.640908067 -0500
@@ -45,6 +45,7 @@

   log.silly('mapToRegistry', 'registry', registry)
+   if (!registry) return cb(null, "http://noregistry/" + name, {})

   var auth = config.getCredentialsByURI(registry)

   // normalize registry URL so resolution doesn't drop a piece of registry URL
